CHANGES

```diff
@@ -5,6 +5,16 @@ Here you can see the full list of changes between nameko versions. Versions
 are in form of *headline.major.minor* numbers. Backwards-compatible changes
 increment the minor version number only.
 
+Version 2.6.0
+-------------
+
+Released: pending
+
+* Environment variables are now interpreted as native YAML data types rather
+  than just strings
+* The WSGI Server now uses an explicit logger so it can be controlled using a
+  logging config
+
```

Comment:
maybe a note that a bunch of backwards compat shims are being dropped (as indicated by their warnings)